src/main/java/dev/jbang/cli/JBang.java

```diff
@@ -265,52 +270,37 @@ public static CommandLine getCommandLine(PrintWriter localout, PrintWriter local
 	}
 
 	public static CommandGroupRenderer getCommandRenderer() {
-		Map<String, List<String>> sections = new LinkedHashMap<>();
-		sections.put("Essentials", asList("run", "build"));
-		sections.put("Editing", asList("init", "edit"));
-		sections.put("Caching", asList("cache", "export", "jdk"));
-		sections.put("Configuration", asList("config", "trust", "alias", "template", "catalog", "app"));
-		sections.put("Other", asList("completion", "info", "version", "wrapper"));
-		Map<String, String> cmds = findExternalCommands();
-		if (!cmds.isEmpty()) {
-			sections.put("External", new ArrayList<>(cmds.keySet()));
-		}
-		CommandGroupRenderer renderer = new CommandGroupRenderer(sections, cmds);
-		return renderer;
+		return new CommandGroupRenderer();
 	}
 
-	private static Map<String, String> findExternalCommands() {
-		Map<String, String> result = new TreeMap<>();
```

Comment:
Yeah my bad if i missed this. We already had this bad experience doing Quarkus cli and removed dynamic help listing and only include for commands that been "installed"